tools/common_lib/src/gemm.cpp

```diff
@@ -12,14 +12,14 @@ std::vector<std::byte> dnnl_gemm_op::gemm(const bindings_t& bindings, opts_t opt
 
     dnnl::memory input_a_memory = [&](const auto& binding)
     {
-        auto ret = dnnl::memory(to_dnnl_mem_desc(binding.shape, binding.layout, binding.dt), engine);
+        auto ret = dnnl::memory(to_dnnl_mem_desc(binding.transposed ? TensorShape{binding.shape.n, binding.shape.c, binding.shape.w, binding.shape.h} : binding.shape, binding.layout, binding.dt), engine);
```

Comment:
For umd dispatcher you swap W and H dimensions and also set new format tag (abdc) - I dont see such format tag here?